contrib/systemd/ocid.service

```diff
@@ -8,7 +8,7 @@ Type=notify
 EnvironmentFile=-/etc/sysconfig/ocid-storage
 EnvironmentFile=-/etc/sysconfig/ocid-network
 Environment=GOTRACEBACK=crash
-ExecStart=/usr/bin/ocid \
+ExecStart=/usr/local/bin/ocid \
```

Comment:
Is there any tweaks necessary to documentation for this change, even if it's only a suggested heads up kind of thing?